doc/api/tls.markdown

```diff
@@ -493,6 +494,24 @@ Example:
 If the peer does not provide a certificate, it returns `null` or an empty
 object.
 
+### tlsSocket.getProtocol()
+
+Returns a string containing the negotiated SSL/TLS protocol version of the
+current connection. `'unknown'` will be returned for connected sockets that have
```

Comment:
Not very easily/reliably as it would probably have to amount to polling. Even at the stage in which an SNI callback is called, the protocol information is already available.